### PR TITLE
Update Italian translation

### PR DESCRIPTION
**Pull request by matteopey**
- 100% coverage for Italian translation.
- Changed a few Italian messages that I think sound better now.

### CODE REVIEW

**Review by whymarrh — APPROVED** [review]
Thanks @matteopey!

(I resolved the conflict in the changelog and made one tiny adjustment to casing.)

File: app/_locales/it/messages.json [hunk 1]
```diff
@@ -71,8 +101,11 @@
   "borrowDharma": {
     "message": "Prendi in presisito con Dharma (Beta)"
   },
+  "browserNotSupported": {
+    "message": "Il tuo Browser non è supportato..."
+  },
   "builtInCalifornia": {
-    "message": "MetaMask è progettato e costruito in California."
+    "message": "MetaMask è progettato e realizzato in California."
```
Comment (Abeseba) [inline app/_locales/it/messages.json]: message realizzto

File: app/_locales/it/messages.json [hunk 2]
```diff
@@ -83,18 +116,42 @@
...
+    "message": "Annulla"
+  },
+  "cancelAttempt": {
+    "message": "Tentativo di Annullamento"
+  },
+  "cancellationGasFee": {
+    "message": "Commissione di Annullamento in Gas"
+  },
+  "cancelN": {
+    "message": "Cancel all $1 transactions"
```
Comment (Abeseba) [inline app/_locales/it/messages.json]: message

**Comment by whymarrh** [comment]
@Abeseba would  you mind opening up a new PR?

**Pull request merged**